sdk/tools/cabman/CCFDATAStorageUnix.cxx

```diff
@@ -0,0 +1,169 @@
+/*
+* COPYRIGHT:   See COPYING in the top level directory
+* PROJECT:     ReactOS cabinet manager
+* FILE         tools/cabman/CCFDATAStorageWindows.cxx
```

Comment:
The file name does not correspond to the actual one.

Reply:
Thank you for your comments, I changed headers of the two files I've added.